README.md

```diff
@@ -1,17 +1,21 @@
-- _All_ the management plane sdks for the V1 Azure API (Azure Service Management) `azure-asm-*` will still be maintained from this repository. We would expect customers using these packages to move to Azure Resource Manager API packages at the earliest.
+We are excited to announce that we are building a new [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js), which includes both Node.js and Browser support! 
 
-**NOTE:** Apart from the noticable changes in **require/import** statements, we have tried our level best to make _lesser_ breaking changes in the API calls.
+The following will help you update your application to the new SDK and related packages.
 
-### Migration
-You can find more information on migrating your scripts from **node** sdk to **JS** sdk over [here](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Migration.md).
+- **Packages**: 
+    - All of the management plane SDKs (`azure-arm-*`) and most of the data plane SDKs (`azure-*`), along with the rollup package [azure](https://www.npmjs.com/package/azure) in this repository, are deprecated as of **July, 2019**. They have been replaced by the [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js) packages.
+    - The new actively managed packages in [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js) support both Node.js and the Browser and ship with TypeScript type definitions.
+    - You will find all equivalent JavaScript packages in `@azure` org on npm for the existing Node.js SDK package. For example, [azure-arm-compute](https://www.npmjs.com/package/azure-arm-compute) Node.js SDK's equivalent JavaScript SDK is [@azure/arm-compute](https://www.npmjs.com/package/@azure/arm-compute). 
+    - All of the management plane SDKs for the V1 Azure API (Azure Service Management) `azure-asm-*` will continue to be maintained from the Azure SDK for Node.js repository. We expect customers using these packages to move to Azure Resource Manager API packages at their earliest convenience.
+- **Breaking Changes**: Apart from the noticable changes in **require/import** statements, we have tried our best to make minimal breaking changes in the API calls.
+- **Migration**: You can find more information on migrating your code from **Azure SDK for Node.js** to the new **Azure SDK for JavaScript** [here](https://github.com/Azure/azure-sdk-for-js/blob/master/documentation/Migration.md).
 
 ## Azure SDK for Node.js
+
+> NOTE: As of July, 2019, this repository applies to the V1 Azure API (Azure Service Management) only. All other packages will be replaced by the [Azure SDK for JavaScript](https://github.com/azure/azure-sdk-for-js). Please see the note above for more information.
```

Comment:
> All other packages will be replaced by the [Azure SDK for JavaScript]

This is not entirely accurate...

From the table at https://github.com/Azure/azure-sdk-for-node/blob/af611c0d8a14882fe941022194ae2786410a8d6e/README.md#azure-service-modules, the below libraries don't seem to have a counterpart
- azure-gallery
- azure-monitoring
- azure-scheduler

@amarzavery Any ideas on why the above don't have a counterpart in the new repo?

Reply:
I reworded to be less specific.